### PR TITLE
Python 3 compatibility (issue #398)

### PR DESCRIPTION
**Pull request by gedeck**
This covers issue #398:

- xrange: replace with range or import from rdkit.six
- removed string.lower

### CODE REVIEW

File: rdkit/Chem/MolSurf.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 from rdkit.Chem.PeriodicTable import numTable
 from rdkit.Chem import Crippen
 from rdkit.Chem import rdPartialCharges, rdMolDescriptors
+from rdkit.six.moves import xrange
```
Comment (greglandrum) [inline rdkit/Chem/MolSurf.py]: Why didn't you just switch to `range` here too?

File: rdkit/ML/Cluster/ClusterVis.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 except ImportError:
   from rdkit.piddle import piddle
 import ClusterUtils
+from rdkit.six.moves import xrange
```
Comment (greglandrum) [inline rdkit/ML/Cluster/ClusterVis.py]: same question: why not just switch this to range too?

Comment (gedeck) [inline rdkit/ML/Cluster/ClusterVis.py]: Same reason for rdkit.ML

File: rdkit/Chem/MolSurf.py (hunk 1 above)
Comment (gedeck) [inline rdkit/Chem/MolSurf.py]: Oh, that is a nasty one. Importing xrange added it to the _descList. So much for quick and dirty fixes.

**Comment by greglandrum** [comment]
Thanks Peter

**Pull request merged**